src/cli/commands/why.js

```diff
@@ -135,10 +136,28 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
 
   const query = await cleanQuery(config, args[0]);
 
-  reporter.step(1, 4, reporter.lang('whyStart', args[0]), emoji.get('thinking_face'));
+  // Steps that will be displayed during the run
+  // These always run so we store them in array so we can keep the length of steps should we add one after these
+  const steps = [
+    {msg: reporter.lang('whyStart', args[0]), emoji: emoji.get('thinking_face'), action: () => {}},
```

Comment:
#nit: since you have separated the `emoji` part, you can simply pass the name of the emoji `emoji: 'thinking_face'` and use `emoji.get(emoji)` below.